rmgpy/data/kinetics/family.py

```diff
@@ -3228,10 +3480,8 @@ def crossValidateOld(self, folds=5, T=1000.0, random_state=1, estimator='rate ru
                 errors[rxn] = np.log(k/krxn)
         
         return errors
-            
-            
-    
-    def simpleRegularization(self, node):
+
+    def simpleRegularization(self, node, templateRxnMap, test=True):
```

Comment:
Could you explain what the test argument does in the docstring?

Reply:
done